tests/test_bids.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import json
+from application import create_app, db
+from application.models.bid import Bid
+from application.models.item import Item
+
+
+class TestUsers(unittest.TestCase):
+    def setUp(self):
+        self.app = create_app('testing')
+        self.test_app = self.app.test_client()
+        with self.app.app_context():
+            db.create_all()
+
+        item = Item(
+            description="Antique Tea set",
+            donor="Demo McDemoFace",
+            price=140.00,
+            status="For Sale",
+            title="Tea Set",
+            category="furniture",
+            charity="Big Cat Rescue",
+            image="img.ul"
+        )
+        with self.app.app_context():
+            db.session.add(item)
+            db.session.commit()
+
+        bid = Bid(
+            bidder_name="Testy McTesterson",
+            bidder_email="testymctesterson@example.com",
+            amount=300.00,
+            street_address="123 This Is My Street Way",
+            city="My City",
+            state="Colorado",
+            zip_code="12345",
+            receipt="img.ul",
+            item_id=1
```

Comment:
Hardcoding this `item_id` in for now and working.  Could be an interpolated Flask syntax similar to Rails to call the `id` of the test-created object?